src/main/java/org/trigon/commands/GearRatioCalculationCommand.java

```diff
@@ -1,19 +1,19 @@
 package org.trigon.commands;
 
-import edu.wpi.first.wpilibj2.command.Command;
-import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.*;
 import org.littletonrobotics.junction.Logger;
 import org.littletonrobotics.junction.networktables.LoggedDashboardBoolean;
 import org.littletonrobotics.junction.networktables.LoggedDashboardNumber;
 
 import java.util.function.DoubleConsumer;
 import java.util.function.DoubleSupplier;
 
-public class GearRatioCalculationCommand extends Command {
+public class GearRatioCalculationCommand extends SequentialCommandGroup {
     private final DoubleSupplier rotorPositionSupplier;
     private final DoubleSupplier encoderPositionSupplier;
     private final DoubleConsumer runGearRatioCalculation;
     private final String subsystemName;
+    private final double backlashAccountabilitySeconds;
```

Comment:
This doesn't seem like a good name to me. How about `overcomeBacklashTimeSeconds`?